draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1820,44 +1820,50 @@ IDNA2003 {{RFC3490}}.
-advertising networks). These third-party servers can use cookies to track the
-user even if the user never visits the server directly. For example, if a user
-visits a site that contains content from a third party and then later visits
-another site that contains content from the same third party, the third party
-can track the user between the two sites.
-
-Given this risk to user privacy, some user agents restrict how third-party
-cookies behave, and those restrictions vary widly. For instance, user agents
-might block third-party cookies entirely by refusing to send Cookie header fields or
-process Set-Cookie header fields during third-party requests. They might take a less
-draconian approach by partitioning cookies based on the first-party context,
-sending one set of cookies to a given third party in one first-party context,
-and another to the same third party in another. Or they might even allow some
-third-party cookies but block others depending on user-agent cookie policy or
-user controls.
-
-This document grants user agents wide latitude to experiment with third-party
-cookie policies that balance the privacy and compatibility needs of their users.
-However, this document does not endorse any particular third-party cookie
-policy.
-
-Third-party cookie blocking policies are often ineffective at achieving their
-privacy goals if servers attempt to work around their restrictions to track
-users. In particular, two collaborating servers can often track users without
-using cookies at all by injecting identifying information into dynamic URLs.
-
-## Cookie policy {#cookie-policy}
+In a Web browsing context, a "third-party" or cross-site cookie is a cookie that is associated with
+content that is served from a different server than the one that hosts the primary resource (that
+is, the HTML). HTML documents can embed content (such as scripts, images, stylesheets, frames) that
+is served from other domains. These third-party resources can use cookies to track the user even if
+they never visit that server directly. For example, if a user visits a site that contains content
+from a third party and then later visits another site that contains content from the same third
+party, the third party can track the user between the two sites.
+
+Because of their inherent privacy issues, most user agents now limit third-party cookies in a
+variety of ways. Some completely block third-party cookies by refusing to process third-party
+Set-Cookie header fields and refusing to send third-party Cookie header fields. Some partition
+cookies based upon the first-party context, so that different cookies are sent depending on the
+site being browsed. Some block cookies based upon user agent cookie policy and/or user controls.
+
+It is RECOMMENDED that user agents implement third-party cookie policies that balance the
+privacy and compatibility needs of their users. However, this document does not yet endorse any
+particular third-party cookie policy.
```

Comment:
This is a bit of a non-statement really.  I'm all for magical thinking, but wouldn't this be better removed?  You already explain that things are unreliable.

If you want to recommend something, how about:

```suggestion
As the situation is fluid, this document does not endorse any particular policy for third-party cookies.  However, user agents SHOULD adopt a policy for third-party cookies that is as restrictive as compatibility constraints permit, thereby providing the best possible privacy.  Any such policy might become stricter as compatibility constraints change over time.

Consequently, resources cannot rely on a specific treatment for cookies in third-party contexts.
```

Reply:
I see it more as air cover than magical thinking. See edits.